contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -964,426 +964,286 @@ contract LegacyPooledStaking is IPooledStaking, MasterAwareV2 {
+    }
+
+    // Abracadabra
+    if (id == 120) {
+      return 39776602220476825000; // 39.776602220476825%
     }
 
     // Premia Finance
-    if (id == 90) {
-      return 13609569933234702000; // 13.609569933234702%
+    if (id == 121) {
+      return 28511599806087503000; // 28.511599806087503%
     }
 
-    // Bunny
-    if (id == 92) {
-      return 34957746687696930000; // 34.957746687696925%
+    // Yearn yvDAI v2
+    if (id == 127) {
+      return 2639172326859299000; // 2.639172326859299%
     }
 
-    // Venus
-    if (id == 93) {
-      return 39982787592238610000; // 39.982787592238616%
+    // Yearn ycrvstETH v2
+    if (id == 129) {
+      return 22160010205633200000; // 22.1600102056332%
     }
 
-    // Thorchain
-    if (id == 94) {
-      return 16340308447531193000; // 16.340308447531193%
+    // Curve sETH LP (eCrv)
+    if (id == 131) {
+      return 3297525661298780400; // 3.29752566129878%
     }
 
-    // Pancakeswap v1
-    if (id == 95) {
-      return 29659851177897560000; // 29.659851177897558%
+    // Idle DAI v4 (idleDAIYield)
+    if (id == 132) {
+      return 50925266656986310000; // 50.92526665698631%
     }
 
-    // Yearn yvDAI v2
-    if (id == 96) {
-      return 2681190691870258400; // 2.6811906918702584%
+    // Idle USDT v4 (idleUSDTYield)
+    if (id == 133) {
+      return 55760635763374930000; // 55.760635763374935%
     }
 
-    // Curve sETH LP (eCrv)
-    if (id == 100) {
-      return 3642751750084770400; // 3.6427517500847704%
+    // Convex stethCrv (cvxstethCrv)
+    if (id == 134) {
+      return 16591126228633247000; // 16.591126228633247%
     }
 
-    // Idle DAI v4 (idleDAIYield)
-    if (id == 101) {
-      return 4831821144747536000; // 4.831821144747536%
+    // Notional Finance v2
+    if (id == 138) {
+      return 14441080249369067000; // 14.441080249369067%
     }
 
-    // Idle USDT v4 (idleUSDTYield)
-    if (id == 102) {
-      return 5089820795846740000; // 5.08982079584674%
+    // OlympusDAO
+    if (id == 139) {
+      return 43265543057180246000; // 43.265543057180246%
     }
 
     // Pool Together v4
-    if (id == 110) {
-      return 13592770586860942000; // 13.592770586860942%
+    if (id == 141) {
+      return 22971686440222584000; // 22.971686440222584%
+    }
+
+    // Origin OUSD
+    if (id == 143) {
+      return 83061669760667070000; // 83.06166976066707%
     }
 
-    // Trader Joe
-    if (id == 111) {
-      return 7995085516794736000; // 7.9950855167947354%
+    // Platypus
+    if (id == 148) {
+      return 5959850680712226500; // 5.9598506807122265%
+    }
+
+    // Bancor v3
+    if (id == 155) {
+      return 50766693682584970000; // 50.76669368258497%
+    }
+
+    // Iron Bank
+    if (id == 157) {
+      return 62666829169994260000; // 62.66682916999426%
     }
     // {V1_PRICES_HELPER_END}
 
-    revert("Invalid product id");
+    // Invalid product Id
+    //    revert("Invalid product id");
+    return type(uint96).max;
```

Comment:
added this but it may be temporary.


The issue here is:


We have deprecated products, for which we don't have a price anymore cause we don't sell them.


Right now i'm skipping their migration.


Unsure here what's the right option
1. migrate with some fixed price 2.6%
2. do not migrate

2. Sounds more sensible because you can't sell those products anymore.

Reply:
It's been settled, we do skip it.